contributing/development/compiling/optimizing_for_size.rst

```diff
@@ -144,6 +144,10 @@ Tools must be disabled in order to use this flag, as the editor is not designed
 to operate without 3D support. Without it, the binary size can be reduced
 by about 15%.
 
+.. note::
+
+    Disabling 3D support also disables all navigation, including 2D navigation.
```

Comment:
```suggestion
    Disabling 3D support also disables all navigation, including 2D navigation
    as it uses 3D navigation internally, there's no dedicated 2D system.
```
Maybe?

Reply:
Probably better to replace 'it' with '2D navigation' to be complete clear.
    Disabling 3D support also disables all navigation, including 2D navigation
    since 2D navigation uses 3D navigation internally, there's no dedicated 2D system.